packages/shared/lib/core/layer-2/interfaces/layer2-native-token.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { IIrc30Metadata } from '@core/wallet'
+
+export interface ILayer2NativeToken {
```

Comment:
What exactly is this an interface for? Is this like wrapped L1 native tokens on L2?

Reply:
I didnt want to overcomplicate the PR making the PR work with the existing `IAsset` because we barely have any information at this point about the asset, so i found it easier using this type until we can format the full information of the asset into something we already have

Reply:
I'll update these types in the next PR to be able to add the assets to the persisted profiles